ha/const.py

```diff
@@ -383,3 +383,8 @@ class K8S_ALERT_RESOURCE_TYPE(Enum, metaclass=EnumListMeta):
     RESOURCE_TYPE_NODE = "node"
     RESOURCE_TYPE_CVG = "cvg"
     RESOURCE_TYPE_DISK = "disk"
+
+class NODE_TYPE(Enum, metaclass=EnumListMeta):
```

Comment:
Class name can be more meaningful.
How about, `class NODE_FUNCTIONAL_TYPES` as in,
https://github.com/Seagate/cortx-ha/pull/691/files#diff-55ccc9a03105a162ccff5059c84a2f6a22be3be8ade6a319921cc6de5337eadaR70-R74

Reply:
If already you defined constants of node types in other PR then I will pick it from there only.

Reply:
fixed.